docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/example-creating-a-dxp-cluster.md

```diff
@@ -1,5 +1,13 @@
 # Example: Creating a DXP Cluster
 
+<!--
+
+# Example: Creating a Simple(?) Cluster
+
+Maybe we should remove DXP from the article title?
+
+-->
+
```

Comment:
@jrhoun I like to leave _DXP_ in the title to distinguish it from clustering other things,like databases, elastic search, ...

How about _Creating a Simple DXP Cluster_?

Reply:
> distinguish it from clustering other things

Makes sense. I think that being the case, maybe just keep the title as is. Or: _Example: Creating a Liferay Cluster_ ?